extensions/packageManager/src/node/index.ts

```diff
@@ -58,6 +119,35 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
         recentlyUsed: recentlyUsed,
       });
     },
+    getFeed: async function (req, res) {
+      // read the list of sources from the config file.
+      const packageSources = [req.query.url];
+
+
+      let combined = [];
+      for (let s = 0; s < packageSources.length; s++) {
```

Comment:
Convert this to a `forEach` instead. Or use `for (const s of packageSources)`

Reply:
- [x]